2012-04-29  Aurelien Jarno  <aurelien@aurel32.net>

        * locales/en_AU: Change date_fmt to match Australian standards.

---
 localedata/locales/en_AU |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/localedata/locales/en_AU
+++ b/localedata/locales/en_AU
@@ -114,7 +114,7 @@
 am_pm       "<U0041><U004D>";"<U0050><U004D>"
 t_fmt_ampm  "<U0025><U0049><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U0070>"
-date_fmt	"<U0025><U0061><U0020><U0025><U0062><U0020><U0025><U0065>/
+date_fmt	"<U0025><U0041><U0020><U0025><U002D><U0065><U0020><U0025><U0042><U0020>/
 <U0020><U0025><U0048><U003A><U0025><U004D><U003A><U0025><U0053><U0020>/
 <U0025><U005A><U0020><U0025><U0059>"
 %  FIXME: found in CLDR
